src/components/Avatar/Avatar.tsx

```diff
@@ -123,7 +123,6 @@ export const Avatar: React.FC<AvatarProps> = (props) => {
               ? {
                   borderRadius: size / 2,
                   height: size,
-                  resizeMode: 'contain',
```

Comment:
@nhannah What do you think about this? This gets rid of the flickering issue on avatar.

Reply:
Looking at this now